src/cargo.js

```diff
@@ -1,7 +1,19 @@
 import { BuildError } from './error';
+import { reverseString } from './util';
 import fse from 'fs-extra';
 import path from 'path';
 
+export const parseDependencies = (data, baseFile) =>
+  data
+    .split('\n')
+    .filter(str => str.startsWith(baseFile))
+    .map(str => str.slice(str.indexOf(': ') + 2))
+    .map(reverseString)
+    .map(str => str.split(/(?: (?!\\))+/))
```

Comment:
Can we avoid reversing each string by using negative lookbehind instead of lookahead here?

Reply:
OK makes sense with your current approach then! :+1: